lib/src/radio-group/radio-group.component.html

```diff
@@ -0,0 +1,35 @@
+<label *ngIf="label" class="radio-group-label"
+    ><b>{{ label }}</b></label
+>
+<div [attr.id]="id" *ngIf="list" class="radio-group {{ className }}" [class.disabled]="disabled">
```

Comment:
In case there is no `className` passed as an input, this is how it's gonna look like `class="radio-group "` with a space at the end. I think if you can create a getter that appends a space with the `className` variable at the end would be a much cleaner solution. Don't you think?

Reply:
I agree, I will use ngClass instead